samples/WOCCatalog/WOCCatalog/UserDefaultsViewController.m

```diff
@@ -30,12 +30,13 @@ - (void)viewDidLoad {
     [super viewDidLoad];
 
     NSUserDefaults* standardDefaults = [NSUserDefaults standardUserDefaults];
-    NSDictionary* dict = @{@"FavoriteWord" : @"Blue"};
+    NSDictionary* dict = @{@"FavoriteWord" : @"Blue", @"FavoriteBool" : [NSNumber numberWithBool:YES]};

```

Comment:
Testing a bool strictly increases our test coverage. I wrote these changes because I couldn't answer the question of whether this scenario had regressed.

---
In reply to: [69818261](https://github.com/Microsoft/WinObjC/pull/612#discussion_r69818261) [](ancestors = 69818261)